web/pdf_thumbnail_viewer.js

```diff
@@ -36,7 +36,6 @@ const THUMBNAIL_SELECTED_CLASS = "selected";
  * @typedef {Object} PDFThumbnailViewerOptions
  * @property {HTMLDivElement} container - The container for the thumbnail
  *   elements.
- * @property {EventBus} eventBus - The application event bus.
```

Comment:
The `EventBus` type import at https://github.com/mozilla/pdf.js/blob/7218bd207fc5b0a8f38a7b9d2a6169e259fc55e4/web/pdf_thumbnail_viewer.js#L17 should also be removed if we remove this line.

Reply:
Thanks, fixed now!

---

Those `@typedef` comments were added solely for the benefit of the TypeScript definitions, since e.g. `gulp typestest` fails without them. So it's pretty annoying that the TypeScript compiler doesn't even warn they become unused :-(